### PR TITLE
Add a bio file for GitHub Teacher

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request adds a bio file for @githubteacher as requested in #3.

@githubstudent I would :heart: if you have some :watch: to put some :eyes: on this and let me know your :thought_balloon:s.

### CODE REVIEW

**Comment by githubteacher** [comment]
This looks good for the most part.  I want to point something out on a particular line though before you merge this in.

File: docs/githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# GitHub Teacher
+
+- **Name:** GitHub Teacher
+- **Occupation:** Teacher
+- **Location:** San Francisco
```
Comment (githubteacher) [inline docs/githubteacher.md]: What if we were more specific here?  There are lots of San Franciscos in the :earth_africa:

File: docs/githubteacher.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# GitHub Teacher
+
+- **Name:** GitHub Teacher
```
Comment (githubteacher) [inline docs/githubteacher.md:3]: Maybe we don't want these three lines bolded.

File: docs/githubteacher.md (hunk 1 above)
Comment (jasonmassey) [inline docs/githubteacher.md]: Agree, please add state and country.

**Comment by githubteacher** [comment]
:+1: This looks good!

:ship: it!

:shipit:!

**Pull request merged**